cmd/nvidia-ctk-installer/main.go

```diff
@@ -123,11 +125,17 @@ func (a app) build() *cli.App {
 			EnvVars:     []string{"TOOLKIT_INSTALL_DIR", "ROOT"},
 		},
 		&cli.StringFlag{
-			Name:        "source-root",
-			Value:       "/",
-			Usage:       "The folder where the required toolkit artifacts can be found",
+			Name:        "toolkit-source-root",
```

Comment:
Do we need to update documentation on this flag name change?

Reply:
This is a new flag and is not publicly accessible. It is internal only.